src/main/java/mongodb/Main.java

```diff
@@ -0,0 +1,220 @@
+
+    // Provide the name of the database and collection you want to use.
+    // If they don't already exist, the driver and Atlas will create them
+    // automatically when you first write data.
+    String dbName = "myDatabase";
+    String collectionName = "recipes";
+
+    // a CodecRegistry tells the Driver how to move data between Java POJOs (Plain Old Java Objects) and MongoDB documents
+    CodecRegistry pojoCodecRegistry = fromRegistries(MongoClientSettings.getDefaultCodecRegistry(),
+            fromProviders(PojoCodecProvider.builder().automatic(true).build()));
+
+    // The MongoClient defines the connection to our MongoDB datastore instance (Atlas) using MongoClientSettings
+    MongoClientSettings settings = MongoClientSettings.builder()
+            .codecRegistry(pojoCodecRegistry)
+            .applyConnectionString(mongoUri).build();
+    MongoClient mongoClient = MongoClients.create(settings);
+
+    // MongoDatabase defines a connection to a specific MongoDB database
+    MongoDatabase database = mongoClient.getDatabase(dbName);
+    // MongoCollection defines a connection to a specific collection of documents in a specific database
+    MongoCollection<Recipe> collection = database.getCollection(collectionName, Recipe.class);
+
+    /*      *** INSERT DOCUMENTS ***
+     *
+     * You can insert individual documents using collection.insert().
+     * In this example, we're going to create 4 documents and then
+     * insert them all in one call with insertMany().
+     */
+
+    List<Recipe> recipes = getRecipes();
+
+    try {
+      InsertManyResult result = collection.insertMany(recipes);
+      System.out.println("Inserted document ids: " + result.getInsertedIds());
+    } catch (MongoException me) {
+      System.err.println("Unable to insert any driver documents due to an error: " + me);
+    }
+
+    /*      *** FIND DOCUMENTS ***
+     *
+     * Now that we have data in Atlas, we can read it. To retrieve all of
+     * the data in a collection, we call find() with an empty filter.
+     */
+
+    try (MongoCursor<Recipe> cursor = collection.find().iterator()) {
+      while (cursor.hasNext()) {
+        System.out.println(cursor.next());
+      }
+    } catch (MongoException me) {
+      System.err.println("Could not find documents due to an error: " + me);
+    }
+
+    // We can also find a single document. Let's find the first document
+    // that has the string "potato" in the Ingredients list. We
+    // use the Filters.eq() method to search for any values in any
+    // ingredients list that match the string "potato":
+
+    Bson findPotato = eq("ingredients", "potato");
+    try {
+      Recipe firstPotato = collection.find(findPotato).first();
+      if (firstPotato == null) {
+        System.out.println("Didn't find any recipes containing 'potato' as an ingredient.");
+      } else {
+        System.out.println(firstPotato);
```

Comment:
Question: this part of the example was a little confusing to me. From Caleb's code, I interpreted it as part of preparation for "Update a document" as nothing was printed: https://github.com/mongodb-university/atlas_starter_dotnet/blob/master/AtlasStarter/Program.cs#L108
Should we modify our examples to print a descriptive message here to let the user what's going on?

Reply:
I think it makes sense to stick to Caleb's example for consistency, though in retrospect printing something out might be an improvement. I'll remove the output in the non-error case here.